language/de/install.php

```diff
@@ -356,15 +357,38 @@
 
 		<br />
 
+		<h1>Wie du ein Update deiner Installation mit dem „Komplettpaket“ durchführst</h1>
+
+		<p>Es wird empfohlen deine Installation mit Hilfe des Komplettpakets zu aktualisieren. Wenn allerdings in deiner Installation die phpBB-Hauptdateien verändert wurden, solltest du stattdessen das „Automatische-Update-Paket“ verwenden, um diese Veränderung nicht zu verlieren. Du kannst deine Installation auch mit den in der INSTALL.html beschriebenen Methoden aktualisieren. Zum Aktualisieren von phpBB mit Hilfe des Komplettpakets musst du folgende Schritte ausführen:</p>
+
+		<ol style="margin-left: 20px; font-size: 1.1em;">
+			<li><strong class="error">Sichere alle Board-Dateien und die Datenbank.</strong></li>
+			<li>Gehe zur <a href="https://www.phpbb.com/downloads/" title="https://www.phpbb.com/downloads/">phpBB.com-Downloadseite</a> und lade das neuste „Komplettpaket“-Archiv runter (<a href="https://www.phpbb.de/go/3.2/downloads">deutschsprachige Downloadseite</a>).</li>
+			<li>Entpacke das Archiv.</li>
+			<li>Entferne (lösche) die <code class="inline">config.php</code>-Datei, und die Verzeichnisse <code class="inline">/images</code>, <code class="inline">/store</code> und <code class="inline">/files</code> <em>aus dem Paket</em> (nicht von deiner Website).</li>
+			<li>Gehe in den Administrations-Bereich, Board-Einstellungen, und stelle sicher, dass prosilver als Standard-Style eingerichtet ist. Wenn nicht, stelle ihn auf prosilver.</li>
```

Comment:
Gehe in den Administrations-Bereich **unter** Board-Einstellungen_(kein Komma)_ und stelle sicher, dass prosilver als Standard-Style eingerichtet ist. Wenn nicht, stelle **den Standard-Style** auf prosilver._(bei ihn ist nicht klar, auf was es sich bezieht)_